src/datastore/event-requests.ts

```diff
@@ -48,7 +48,7 @@ export async function* getRawEventRequests(
           id bigint PRIMARY KEY,
           receive_timestamp timestamptz NOT NULL,
           event_path text NOT NULL,
-          payload jsonb NOT NULL
+          payload text NOT NULL
```

Comment:
Changed this to `text` because `jsonb` takes around 25% additional overhead for parsing, and we only never use the `payload` jsonb query features.